static/app/gettingStartedDocs/javascript/react.tsx

```diff
@@ -69,6 +71,27 @@ export const steps = ({
       },
     ],
   },
+  {
+    language: 'html',
+    type: StepType.CONFIGURE_SOURCE_MAPS,
+    configurations: [
+      {
+        description: t(
+          'The easiest way to configure upload Source Maps with Webpack is by using the Sentry Wizard.'
+        ),
+        code: `npx @sentry/wizard@latest -i sourcemaps`,
+        additionalInfo: (
+          <Alert type="info" showIcon noBottomSpacing>
+            {tct('Prefer to set up Source Maps manually? [docsLink:Read the docs].', {
+              docsLink: (
+                <ExternalLink href="https://docs.sentry.io/platforms/javascript/sourcemaps/uploading/webpack/#manual-setup" />
```

Comment:
We should link to the generic one, not the webpack one.
Probably just here:
https://docs.sentry.io/platforms/javascript/sourcemaps/uploading/